Manifest.toml

```diff
@@ -0,0 +1,130 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Do we have to version the `Manifest.toml` file?

Reply:
I will deleted when the PR is ready to merge. Makes it easier to develop in an environment.